src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -41,17 +41,20 @@ public void RelativeSearchPath_Is_Null()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Corefx has limitations to build a UWP executable that can be launched directly using Process.Start")]
         public void TargetFrameworkTest()
         {
-            string targetFrameworkName = ".NETCoreApp,Version=v2.1";
-            if (PlatformDetection.IsInAppContainer)
+            const int ExpectedExitCode = 0;
+            const string AppName = "TargetFrameworkNameTestApp.dll";
+            var psi = new ProcessStartInfo();
+            psi.FileName = RemoteExecutor.HostRunner;
+            psi.Arguments = $"{AppName} {ExpectedExitCode}";
+
+            using (Process p = Process.Start(psi))
```

Comment:
Nit: you don't need the psi... you can just pass the name and args directly to start

Reply:
I'll leave this as is for now as we have multiple places where we are following this pattern.

Reply:
For reference can you link to some?

Reply:
Sorry, I never followed on this and just saw it. Couldn't remember which places I meant back then.